### PR TITLE
Extend hasGeometry to allow for GeometryCollections

### PR DESCRIPTION
**Pull request by nicholascar**
This PR extends the range of `hasGeometry` to allow for the union of `Geometry` and GeometryCollection`. This change is requested to allow for more sophisticated representations of the spatiality of `Features`. Ultimately, this change will allow for a form of fuzzy (uncertain) spatiality by using `Geometry Collections` of related `Geometry` objects.

This PR also:

* updates the version numbers of GeoSPARQL in this branch from 1.1 to 1.3
* fixes some small ASCIIDOC formatting and referencing issues

### CODE REVIEW

**Review by situx — APPROVED** [review]
Looks good to me!

File: spec/sections/09-core.adoc [hunk 1]
```diff
@@ -459,7 +461,13 @@ geo:hasGeometry
...
     rdfs:isDefinedBy geo: ;
     rdfs:domain geo:Feature ;
-    rdfs:range geo:Geometry ;    
+    rdfs:range
+		[
+			owl:unionOf (
+				:Geometry
+				:GeometryCollection
+			)
+		] ;
```
Comment (mperry455) [inline spec/sections/09-core.adoc:470]: I think using a union for rdfs:range here takes us out of OWL2 RL profile. We probably need a larger discussion for this one.

Comment (nicholascar) [reply spec/sections/09-core.adoc:470]: Are you sure that `range -> UnionClassExpression` isn't allowed in RL?

My reading of the [OWL RL Class Expressions](https://www.w3.org/TR/2012/REC-owl2-profiles-20121211/#Class_Expressions_3) grammar is that class union expressions_are_ allowed.

The grammar line for RL's object property allowances states

```
subObjectUnionOf := 'ObjectUnionOf' '(' subClassExpression subClassExpression { subClassExpression } ')'
```

indicates that an RL-legal "object union of" can be declared for (arbitrary) subclasses expressions, with subclass expressions themselves being

```
subClassExpression :=
    Class other than owl:Thing |
    subObjectIntersectionOf | subObjectUnionOf | ObjectOneOf |
    subObjectSomeValuesFrom | ObjectHasValue |
    DataSomeValuesFrom | DataHasValue
```

which means union on `ClassX ClassY` would be fine.

Comment (mperry455) [reply spec/sections/09-core.adoc:470]: Looks like UnionClassExpressions are allowed in general, but the spec mentions this restriction.

> OWL 2 RL axioms about property expressions are as in the structural specification [[OWL 2 Specification](https://www.w3.org/TR/2012/REC-owl2-profiles-20121211/#ref-owl-2-specification)], the only differences being that class expressions in property domain and range axioms are restricted to superClassExpression, and that the use of reflexive properties is disallowed.

`ObjectPropertyRange := 'ObjectPropertyRange' '(' axiomAnnotations ObjectPropertyExpression superClassExpression ')'`

From what I can tell, superClassExpression does not include these union expressions.
```
superClassExpression :=
    Class other than owl:Thing |
    superObjectIntersectionOf | superObjectComplementOf |
    superObjectAllValuesFrom | ObjectHasValue | superObjectMaxCardinality |
    DataAllValuesFrom | DataHasValue | superDataMaxCardinality
```
It seems like we may need a concrete superclass of Geometry and GeometryCollection, which feels kind of awkward.

Comment (situx) [reply spec/sections/09-core.adoc:470]: If we need such classes my suggestion would be: geo:{SpatialObject|Feature|Geometry}Manifestation whereas geo:SpatialObjectManifestation is the superclass of the other two and geo:Geometry and geo:GeometryCollection are subclasses of geo:GeometryManifestation.

But I agree with Matt. Would be good if we could avoid this class hierarchy.

**Comment by situx** [comment]
GeoSPARQL Meeting 26th June:
- Leave the range of geo:hasGeometry unchanged
- Make geo:SpatialObjectCollection subClassOf geo:SpatialObject
- Make geo:FeatureCollection subClassOf geo:Feature
- - Make geo:GeometryCollection subClassOf geo:Geometry

**Comment by nicholascar** [comment]
All changes requested in https://github.com/opengeospatial/ogc-geosparql/pull/525#issuecomment-2198367182 made, plus a bunch of style improvements and minor updates

**Review by mperry455 — APPROVED** [review]

**Review by jabhay — APPROVED** [review]
Looks good to me.

**Pull request merged**